airflow/api_connexion/schemas/xcom_schema.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from marshmallow import post_dump
+from marshmallow_sqlalchemy import SQLAlchemySchema, auto_field
+
+from airflow.models import XCom
+
+
+class XComCollectionItemSchema(SQLAlchemySchema):
+
+    class Meta:
+        """ Meta """
+        model = XCom
+
+    COLLECTION_NAME = 'xcom_entries'
+    FIELDS_FROM_NONE_TO_EMPTY_STRING = ['key', 'task_id', 'dag_id']
```

Comment:
These fields are primary keys, so they must always be value. Why do you want them to be empty?

Reply:
yep realised, will remove them in my next draft